kernel/kernel-api/src/main/java/io/delta/kernel/types/StructField.java

```diff
@@ -95,6 +162,11 @@ public boolean isNullable() {
     return nullable;
   }
 
+  /** @return the list of type changes for this field */
```

Comment:
Is it a list because the filed can undergo more than one type change? Also is the last one the latest one?

Reply:
yes, updated docs.